pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/Main.scala

```diff
@@ -1,37 +1,78 @@
 package uk.ac.wellcome.platform.transformer.sierra
 
+import scala.concurrent.ExecutionContext
+
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
+
+import org.scanamo.DynamoFormat
+
 import com.typesafe.config.Config
 import uk.ac.wellcome.json.JsonUtil._
-import uk.ac.wellcome.messaging.sns.NotificationMessage
-import uk.ac.wellcome.messaging.typesafe.{MessagingBuilder, SQSBuilder}
 import uk.ac.wellcome.models.transformable.SierraTransformable
 import uk.ac.wellcome.models.transformable.SierraTransformable._
 import uk.ac.wellcome.models.work.internal.TransformedBaseWork
 import uk.ac.wellcome.platform.transformer.sierra.services.{
+  EmptyMetadata,
   HybridRecordReceiver,
   SierraTransformerWorkerService
 }
-import uk.ac.wellcome.storage.typesafe.S3Builder
 import uk.ac.wellcome.typesafe.WellcomeTypesafeApp
 import uk.ac.wellcome.typesafe.config.builders.AkkaBuilder
 
-import scala.concurrent.ExecutionContext
+import uk.ac.wellcome.bigmessaging.typesafe.BigMessagingBuilder
+import uk.ac.wellcome.messaging.sns.NotificationMessage
+import uk.ac.wellcome.messaging.typesafe.SQSBuilder
+import uk.ac.wellcome.messaging.sns.SNSConfig
+
+import uk.ac.wellcome.storage.store.dynamo.{DynamoHashStore, DynamoHybridStore}
+import uk.ac.wellcome.storage.store.s3.S3TypedStore
+import uk.ac.wellcome.storage.store.HybridIndexedStoreEntry
+import uk.ac.wellcome.storage.{ObjectLocation, ObjectLocationPrefix}
+import uk.ac.wellcome.storage.typesafe.{DynamoBuilder, S3Builder}
+import uk.ac.wellcome.storage.dynamo.DynamoHashEntry
+import uk.ac.wellcome.storage.streaming.Codec._
 
 object Main extends WellcomeTypesafeApp {
+
+  type IndexEntry = HybridIndexedStoreEntry[ObjectLocation, EmptyMetadata]
+
+  type HashEntry = DynamoHashEntry[String, Int, IndexEntry]
+
   runWithConfig { config: Config =>
+    // TODO: from where do we get the correct values for this?
+    val objectLocationPrefix = ObjectLocationPrefix("namespace", "path")
+    val dynamoConfig =
+      DynamoBuilder.buildDynamoConfig(config, namespace = "namespace")
```

Comment:
Any ideas where I get these from? The `Config`? :thinking:

Reply:
Where you get what from? The prefix?

Reply:
The `namespace` and `path` strings

Reply:
These are presumably needed for accessing the correct S3 location? Needed by `new DynamoHybridStore` below.

Reply:
In relation to the discussion below, is the only way we know the `ObjectLocation` from the incoming message? If so then the strategy of using the hybrid store rather than S3 directly is not going to work :see_no_evil:

Reply:
I think the value of `ObjectLocationPrefix` in this case is stable and can be provided via configuration.